templates/.circleci/config.yml

```diff
@@ -1,41 +1,38 @@
-version: 2.0
+version: 2.1
+orbs:
+  browser-tools: circleci/browser-tools@1.2.5
 jobs:
   build:
     docker:
-      - image: circleci/ruby:__ruby_version__-stretch-node-browsers
+      - image: cimg/ruby:__ruby_version__-browsers
```

Comment:
Is this valid? I have never seen `-browsers` without a prefix

Reply:
the new `cimg/` images allow `-browsers` as a standalone. This is the template file, so `__ruby_version__` will be replaced with the app's actual ruby version when `gnarails new` is called.